.github/actions/constellation_create/gcp-logs.sh

```diff
@@ -3,49 +3,19 @@
 set -euo pipefail
 shopt -s inherit_errexit
 
-pushd constellation-terraform
-controlInstanceGroup=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_control_plane") |
-    .resources[0].values.base_instance_name'
-)
-workerInstanceGroup=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_worker") |
-     .resources[0].values.base_instance_name'
-)
-zone=$(
-  terraform show -json |
-    jq -r .'values.root_module.child_modules[] |
-    select(.address == "module.instance_group_control_plane") |
-     .resources[0].values.zone'
-)
-popd
+echo "Using Zone: ${1}"
+echo "Using Constellation UID: ${2}"
 
-controlInstances=$(
-  gcloud compute instance-groups managed list-instances "${controlInstanceGroup##*/}" \
-    --zone "${zone}" \
-    --format=json |
-    jq -r '.[] | .instance'
+allInstances=$(
+  gcloud compute instances list \
+    --filter="labels.constellation-uid=${2}" \
+    --format=json | yq '.[] | .name'
 )
-workerInstances=$(
-  gcloud compute instance-groups managed list-instances "${workerInstanceGroup##*/}" \
-    --zone "${zone}" \
-    --format=json |
-    jq -r '.[] | .instance'
-)
-
-allInstances="${controlInstances} ${workerInstances}"
-
-printf "Fetching logs for %s and %s\n" "${controlInstances}" "${workerInstances}"
 
 for instance in ${allInstances}; do
-  shortName=${instance##*/}
   printf "Fetching for %s\n" "${shortName}"
```

Comment:
```suggestion
  printf "Fetching for %s\n" "${instance}"
```
This has to be update as well of course